align/main.py

```diff
@@ -49,8 +49,9 @@ def schematic2layout(netlist_dir, pdk_dir, netlist_file=None, subckt=None, worki
     for netlist in netlist_files:
         logging.info(f"READ file: {netlist} subckt={subckt}, flat={flatten}")
         # Generate hierarchy
-        (working_dir / '1_topology').mkdir(exist_ok=True)
-        primitives = generate_hierarchy(netlist, subckt, working_dir / '1_topology', flatten, unit_size_mos , unit_size_cap)
+        topology_dir = working_dir / '1_topology'
+        topology_dir.mkdir(exist_ok=True)
+        primitives = generate_hierarchy(netlist, subckt, topology_dir, flatten, unit_size_mos , unit_size_cap)
         # Generate primitives
         primitive_dir = (working_dir / '2_primitives')
         primitive_dir.mkdir(exist_ok=True)
```

Comment:
Should use primitive_dir two lines below instead of constructing the path again.

Reply:
Done !